app/activeplan/homepage/get.controller.js

```diff
@@ -1,13 +1,25 @@
 const { logger } = require('../../../common/logging/logger')
+const { getSentencePlanComments } = require('../../../common/data/sentencePlanningApi')
+const { getCommentText } = require('../../../common/utils/getCommentText')
 
 const getHomepage = async (req, res) => {
   const {
-    session: { planStarted = false },
+    session: { planStarted = false, 'x-auth-token': token },
     params: { planId, id },
   } = req
   try {
     delete req.session.planStarted
-    return res.render(`${__dirname}/index`, { planId, id, planStarted })
+    const renderDetails = {}
```

Comment:
Ok so this is ties in with the previous MR. I don't like the creation of the temporary holding object. Could you not just return the `renderDetails` Object, but define it with the other values, i.e. `const renderDetails = { planId, id, planStarted}`.

Or just define them in the return which I'm fine with, but some coding styles don't like such things:
```
return res.render(`${__dirname}/index`, { 
  planId, 
  id, 
  planStarted,
  contactArrangements: getCommentText(comments, 'LIAISON_ARRANGEMENTS'),
  diversity: getCommentText(comments, 'YOUR_RESPONSIVITY'),
  decisions: getCommentText(comments, 'YOUR_SUMMARY')
  needToKnow: getCommentText(comments, 'THEIR_RESPONSIVITY')
  comments: getCommentText(comments, 'THEIR_SUMMARY')
})
```